### PR TITLE
[magnum] fix cross compile

### PR DESCRIPTION
**Pull request by autoantwort**
Otherwise fails with
```
[42/46] cmd.exe /C "cd /D C:\v\vcpkg4\buildtrees\magnum\src\v2020.06-4ccec3ac49.clean\src\MagnumPlugins\AnyImageConverter && C:\v\vcpkg4\vcpkg_installed\arm64-uwp\tools\corrade\corrade-rc.exe AnyImageConverter C:/v/vcpkg4/buildtrees/magnum/arm64-uwp-dbg/src/MagnumPlugins/AnyImageConverter/resources_AnyImageConverter.conf C:/v/vcpkg4/buildtrees/magnum/arm64-uwp-dbg/src/MagnumPlugins/AnyImageConverter/resource_AnyImageConverter.cpp"
FAILED: src/MagnumPlugins/AnyImageConverter/resource_AnyImageConverter.cpp C:/v/vcpkg4/buildtrees/magnum/arm64-uwp-dbg/src/MagnumPlugins/AnyImageConverter/resource_AnyImageConverter.cpp 
cmd.exe /C "cd /D C:\v\vcpkg4\buildtrees\magnum\src\v2020.06-4ccec3ac49.clean\src\MagnumPlugins\AnyImageConverter && C:\v\vcpkg4\vcpkg_installed\arm64-uwp\tools\corrade\corrade-rc.exe AnyImageConverter C:/v/vcpkg4/buildtrees/magnum/arm64-uwp-dbg/src/MagnumPlugins/AnyImageConverter/resources_AnyImageConverter.conf C:/v/vcpkg4/buildtrees/magnum/arm64-uwp-dbg/src/MagnumPlugins/AnyImageConverter/resource_AnyImageConverter.cpp"
This version of C:\v\vcpkg4\vcpkg_installed\arm64-uwp\tools\corrade\corrade-rc.exe is not compatible with the version of Windows you're running. Check your computer's system information and then contact the software publisher.
```

### CODE REVIEW

File: ports/magnum/vcpkg.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
   "homepage": "https://magnum.graphics/",
   "license": null,
   "dependencies": [
+    {
+      "name": "corrade",
+      "host": true,
+      "features": [
+        "utility"
+      ]
+    },
```
Comment (jimwang118) [inline ports/magnum/vcpkg.json:15]: The newly added dependencies exist below. If you only add `host:ture` to the existing dependencies, the repair can be completed.

**Review by jimwang118 — CHANGES_REQUESTED** [review]

**Comment by jimwang118** [comment]
Note: I will be converting your PR to draft status. The above suggested changes are only recommendations. If you are willing to adopt them, please click "ready for review" after making the modifications. If you do not wish to make any changes, please click "ready for review" directly. That way, I can be aware that you've responded since you can't modify the tags.

File: ports/magnum/vcpkg.json (hunk 1 above)
Comment (autoantwort) [reply ports/magnum/vcpkg.json:15]: But the dependency is also needed for host: false, therefore I need two entries.

**Review by jimwang118 — APPROVED** [review]

**Comment by jimwang118** [comment]
Thank you for submitting this PR. Our CI does not have a testing environment. Please help confirm that there are no problems with your local testing.

**Comment by autoantwort** [comment]
I have tested the build on x64-uwp, arm64-uwp, arm64-windows.

**Pull request merged**